docs/advanced/api_examples.yml

```diff
@@ -282,6 +282,31 @@
 
 -
   id:
+    "create container with links"
+
+  doc: |
+    Create a new container with links to one or more other containers.
+    See https://docs.clusterhq.com/en/latest/advanced/configuration.html for more information on Flocker's implementation of container links.
```

Comment:
That's a bit of a vague link (there's lot of stuff in configuration) and we're going to be de-emphasizing the configuration language going forward. I recommend just writing some stuff here that explains what's going on in the context of this particular example. "Presuming we have another container exposed on port 54320, this will setup some env varaibles (link to docker docs) like so" and so on.